scripts/apps/users/views/user-preferences.html

```diff
@@ -423,7 +423,7 @@ <h4 class="sd-heading sd-text-align--left sd-text--sans sd-heading--h4 sd-margin
                     </div>
                 </li>
 
-                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start"></li>
+                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start">
```

Comment:
do the last 2 empty `<li>` tags in this file server any purpose? If not, drop them

Reply:
done